packages/key-did-resolver/src/__tests__/ed25519.test.ts

```diff
@@ -1,12 +1,12 @@
 import varint from 'varint'
-import multibase from 'multibase'
+import { base58btc } from 'multiformats/bases/base58'
 import * as mapper from '../ed25519'
 
 describe('Ed25519 mapper', () => {
   it('successfully resolves the document from did', async () => {
     const id = 'z6MktvqCyLxTsXUH1tUZncNdVeEZ7hNh7npPRbUU27GTrYb8'
 
-    const multicodecPubKey = multibase.decode(id)
+    const multicodecPubKey = base58btc.decode(id)
```

Comment:
PLEASE CHECK

The old multibases used this https://github.com/multiformats/js-multibase/blob/master/src/constants.js to determine the encoding. 

This does not exist in multiformats. I am assuming its base58btc as the tests begin with a z. Not sure if this always be the case.